salt/returners/rawfile_json.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+'''
+Take data from salt and "return" it into a raw file containing the json, with one line per event.
+
+Add the following to the minion or master configuration file.
+
+... code-block: yaml
+
+    rawfile_json.filename: <path_to_output_file>
+
+Default is /var/log/salt/events.
+
+Common use is to log all events on the master. This can generate a lot of
+noise, so you may wish to configure batch processing and/or configure the
+event_return_whitelist or event_return_blacklist to restrict the events
+that are written.
+'''
+
+# Import python libs
+from __future__ import absolute_import, print_function, with_statement
+import logging
+import json
+
+import salt.returners
+
+log = logging.getLogger(__name__)
+
+# Define the module's virtual name
+__virtualname__ = 'rawfile_json'
+
+
+def __virtual__():
+    return __virtualname__
+
+
+def _get_options(ret):
+    '''
+    Returns options used for the carbon returner.
```

Comment:
Good eye. I'm new to Python. Fixed in latest set of patches. I appreciate the feedback.